internal/cmd/loadbalancer/list.go

```diff
@@ -167,3 +171,72 @@ var ListCmd = base.ListCmd{
 		return loadBalancerSchemas
 	},
 }
+
+func loadBalancerHealth(l *hcloud.LoadBalancer) string {
+	healthyCount := 0
+	unhealthyCount := 0
+	unknownCount := 0
+
+	for _, lbTarget := range l.Targets {
+		switch loadBalancerTargetHealth(&lbTarget) {
+		case string(hcloud.LoadBalancerTargetHealthStatusStatusHealthy):
+			healthyCount++
+
+		case string(hcloud.LoadBalancerTargetHealthStatusStatusUnhealthy):
+			unhealthyCount++
+
+		case "mixed":
+			return "mixed"
+
+		default:
+			unknownCount++
+		}
+	}
+
+	switch len(l.Targets) {
+	case healthyCount:
+		return string(hcloud.LoadBalancerTargetHealthStatusStatusHealthy)
+
+	case unhealthyCount:
+		return string(hcloud.LoadBalancerTargetHealthStatusStatusUnhealthy)
+
+	case unknownCount:
+		return string(hcloud.LoadBalancerTargetHealthStatusStatusUnknown)
+
+	default:
+		return "mixed"
+	}
+}
+
+func loadBalancerTargetHealth(t *hcloud.LoadBalancerTarget) string {
+	healthyCount := 0
+	unhealthyCount := 0
+	unknownCount := 0
+
+	for _, targetHealth := range t.HealthStatus {
+		switch targetHealth.Status {
+		case hcloud.LoadBalancerTargetHealthStatusStatusHealthy:
+			healthyCount++
+
+		case hcloud.LoadBalancerTargetHealthStatusStatusUnhealthy:
+			unhealthyCount++
+
+		default:
+			unknownCount++
+		}
+	}
+
+	switch len(t.HealthStatus) {
+	case healthyCount:
+		return string(hcloud.LoadBalancerTargetHealthStatusStatusHealthy)
+
+	case unhealthyCount:
+		return string(hcloud.LoadBalancerTargetHealthStatusStatusUnhealthy)
+
+	case unknownCount:
+		return string(hcloud.LoadBalancerTargetHealthStatusStatusUnknown)
+
+	default:
+		return "mixed"
+	}
+}
```

Comment:
Absolutely fair point! 
It makes the code more concise.

To clarify: I did consider this method as well, but opted to separate the functions to avoid nesting for-loops. It's just something I try to avoid.